platform/viewer/pluginConfig.js

```diff
@@ -0,0 +1,112 @@
+/**
```

Comment:
Renamed to js to allow comments and easier syntax.
There are lots of changes in the file compared to the json file, but almost none of it is relevant - you can just return the json contents and it all works.  YOu can also just delete this file and use an existing json file and it also still all works.